code/controllers/configuration/entries/comms.dm

```diff
@@ -20,9 +28,25 @@
 /datum/config_entry/keyed_list/cross_server/ValidateListEntry(key_name, key_value)
 	return key_value != "byond:\\address:port" && ..()
 
+/datum/config_entry/keyed_list/insecure_cross_server
+	key_mode = KEY_MODE_TEXT
+	value_mode = VALUE_MODE_TEXT
+	protection = CONFIG_ENTRY_LOCKED
+
+/datum/config_entry/keyed_list/insecure_cross_server/ValidateAndSet(str_val)
+	. = ..()
+	if(.)
+		var/list/newv = list()
+		for(var/I in config_entry_value)
+			newv[replacetext(I, "+", " ")] = config_entry_value[I]
+		config_entry_value = newv
+
+/datum/config_entry/keyed_list/insecure_cross_server/ValidateListEntry(key_name, key_value)
+	return key_value != "byond:\\address:port" && ..()
```

Comment:
You do forward slashes in the config files.

Reply:
oops, that was an oversight because the configs (incorrectly) used to have \\